src/worldpatch/resource_manager.zig

```diff
@@ -156,27 +156,27 @@ pub const WorldPatchManager = struct {
     }
 
     pub fn registerRequester(self: *WorldPatchManager, id: IdLocal) RequesterId {
-        const requester_id = @intCast(u8, self.requesters.items.len);
+        const requester_id = @as(u8, @intCast(self.requesters.items.len));
```

Comment:
hmmm was this done in resource_manager? looks like world_patch_manager code..

Reply:
I think this was me running `zig fmt` on the project, since I don't have it integrated in VSCode on this machine

Reply:
Fair enough! :)